src/pages/BuyGMX/constants.ts

```diff
@@ -14,26 +14,107 @@ type Exchange = {
 
 export const EXTERNAL_LINKS = {
   [ARBITRUM]: {
-    bungee: `https://multitx.bungee.exchange/?toChainId=42161&toTokenAddress=${ARBITRUM_GMX}`,
-    banxa: "https://gmx.banxa.com/?coinType=ETH&fiatType=USD&fiatAmount=500&blockchain=arbitrum",
-    o3: "https://o3swap.com/",
     networkWebsite: "https://arbitrum.io/",
     buyGmx: {
-      banxa: "https://gmx.banxa.com/?coinType=GMX&fiatType=USD&fiatAmount=500&blockchain=arbitrum",
       uniswap: `https://app.uniswap.org/#/swap?inputCurrency=ETH&outputCurrency=${ARBITRUM_GMX}`,
     },
   },
   [AVALANCHE]: {
-    bungee: `https://multitx.bungee.exchange/?toChainId=43114&toTokenAddress=${AVALANCHE_GMX}`,
-    banxa: "https://gmx.banxa.com/?coinType=AVAX&fiatType=USD&fiatAmount=500&blockchain=avalanche",
     networkWebsite: "https://www.avax.network/",
     buyGmx: {
-      banxa: "https://gmx.banxa.com/?coinType=GMX&fiatType=USD&fiatAmount=500&blockchain=avalanche",
       traderjoe: `https://traderjoexyz.com/trade?outputCurrency=${AVALANCHE_GMX}`,
     },
   },
 };
 
+export const FIAT_GATEWAYS: Exchange[] = [
+  {
+    name: "Binance Connect",
+    icon: "ic_binance.svg",
+    networks: [ARBITRUM, AVALANCHE],
+    link: "https://www.binancecnt.com/en/buy-sell-crypto",
```

Comment:
hm, maybe not duplicating supported networks in networks and links?

I mean, we could leave only the links field and check Object.keys(links).includes(chainId)

Reply:
Done!